presto-main/src/main/java/com/facebook/presto/server/CoordinatorModule.java

```diff
@@ -251,6 +253,7 @@ protected void setup(Binder binder)
         binder.bind(DispatchManager.class).in(Scopes.SINGLETON);
         binder.bind(FailedDispatchQueryFactory.class).in(Scopes.SINGLETON);
         binder.bind(DispatchExecutor.class).in(Scopes.SINGLETON);
+        newOptionalBinder(binder, PrerequisiteManager.class);
```

Comment:
A better way to model this is through a Plugin.  i.e., one should just be able to specify a Plugin, which may or may not be packaged with the core Presto distribution, to override this in a customizeable way.  See examples of `TempStorageFactory`, `FunctionNamespaceManagerFactory`, etc. in `PluginManager`.